add compat data for PermissionStatus

gsouquet opened the pull request:
https://developer.mozilla.org/docs/Web/API/PermissionStatus

connorshea commented on api/PermissionStatus.json:
```diff
@@ -0,0 +1,106 @@
+{
+  "api": {
+    "AbortSignal": {
```
This isn't the right API name :)

connorshea commented:
This doesn't include [status](https://developer.mozilla.org/en-US/docs/Web/API/PermissionStatus/status), though that might be intentional since it's deprecated? I'd still say it's useful since some browsers still seem to implement it.

gsouquet commented:
Thank @connorshea for the review, fixed the API name.
Regarding `status` it is referenced as an `alternative_name` in `state`. The spec has evolved and the Chrome implementation was initially with `status` and moved away from it.
Hope that makes sense

ddbeck commented:
This looks good. Thanks for the PR, @gsouquet, and reviewing it, @connorshea! 🎉
